librosa/effects.py

```diff
@@ -660,3 +661,65 @@ def preemphasis(y, coef=0.97, zi=None, return_zf=False):
         return y_out, z_f
 
     return y_out
+
+
+def deemphasis(y, coef=0.95, zi=None):
+    """De-emphasize an audio signal with the inverse operation of preemphasis():
+
+    If y~ = preemphasis(y, coef=coef, zi=zi)
+        y[i] -> y~[i] + coef * y[i-1]
+        y = deemphasis(y~, coef=coef, zi=zi)
+
+
+    Parameters
+    ----------
+    y : np.ndarray
+        Audio signal
+
+    coef : positive number
+        Pre-emphasis coefficient.  Typical values of ``coef`` are between 0 and 1.
+
+        At the limit ``coef=0``, the signal is unchanged.
+
+        At ``coef=1``, the result is the first-order difference of the signal.
+
+        The default (0.97) matches the pre-emphasis filter used in the HTK
+        implementation of MFCCs [#]_.
+
+        .. [#] http://htk.eng.cam.ac.uk/
+
+    zi : number
+        Initial filter state. If inverting a previous preemphasis(), the same value should be used.
+         When making successive calls to non-overlapping
```

Comment:
incomplete sentence?

Reply:
Fixed in https://github.com/librosa/librosa/pull/1293/commits/5771637b181351a885e2fee266c0a03a397b4e09